docs/data/material/migration/migrating-to-v6/migrating-to-pigment-css.md

```diff
@@ -173,14 +186,14 @@ Integrating Pigment CSS with Material UI requires you to configure the theme t
 Add the following code to your [Next.js](#nextjs) or [Vite](#vite) config file:
 
 ```diff
-+import { extendTheme } from '@mui/material';
++import { createTheme } from '@mui/material';
 
 +const pigmentConfig = {
-+  theme: extendTheme(),
++  theme: createTheme(…parameters if any),
```

Comment:
Maybe this is clear enough?

```suggestion
+  theme: createTheme({}),
```

Reply:
I think `createTheme({})` is not what I'm looking for. My intention is to say "the theme overrides that you are using go here", cc @samuelsycamore any thought on how to make this better?

Reply:
Should we include `transformLibraries: ['@mui/material'],` here as well?

Reply:
Done

Reply:
Added.

Reply:
@siriwatknp I think you forgot to push the change.